server/app.js

```diff
@@ -1,6 +1,11 @@
 const express = require('express');
 const path = require('path');
-const db = require('../db/db.js');
+const db = require('../db/mysql.js');
+
```

Comment:
you have two variables called db.

Reply:
not sure why github doesn't have my latest change?